src/Integration.Vsix/SonarLintTagger/IssueToIssueMarkerConverter.cs

```diff
@@ -58,7 +62,8 @@ public IssueMarker Convert(IAnalysisIssue issue, ITextSnapshot textSnapshot)
 
             var span = issueSpanCalculator.CalculateSpan(issue, textSnapshot);
 
-            return span == null ? null : new IssueMarker(issue, span.Value);
+            var issueViz = issueVizConverter.Convert(issue);
+            return span == null ? null : new IssueMarker(issueViz, span.Value);
```

Comment:
Just for my understanding: 
The issue viz will always be created only if there is a span.
That span can later be changed to a null if the code changes, but it will never be null at the beginning, correct?

Reply:
Currently yes, since that's what the previous code did. That will probably change in the future when I merge the issue marker and an issue viz converters.